<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServer</class>
 <widget class="QWidget" name="TcpServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>623</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="serverInfoTitle">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ServerInfo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="serverInfoList"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="selectClientTitle">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>指定客户端</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="descriptorInput"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请输入你要发送的消息：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="sendToClientMessage">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendMessageButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="onlineClientStatusTitle">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>当前在线用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="currentClientNum">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="onlineClientStatus">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <column>
        <property name="text">
         <string>IP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Port</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Descriptor</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>UserID</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="IPtitle">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipInput"/>
     </item>
     <item>
      <widget class="QLabel" name="portInputTitle">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>端口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portInput">
       <property name="text">
        <string>12345</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StartServer">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
